packages/terra-tabs/src/common-tabs/CommonTabs.jsx

```diff
@@ -50,6 +50,23 @@ const propTypes = {
    * Parameters: 1. Event 2. Selected pane's key
    */
   onChange: PropTypes.func,
+  /**
+   * Callback function when add button selection has changed.
+   * Parameters: 1. Event 2. Selected pane's key
+   */
+  onSelectAddButton: PropTypes.func,
+  /**
+   * The label to set on the add icon element.
+   */
+  ariaLabelAddTab: PropTypes.string,
+  /**
+   * Callback function triggered when a tab is closing. It receives the itemKey associated with the closing tab.
+   */
+  onClosingkey: PropTypes.func.isRequired,
+  /**
+   * Callback function when a tab is closing. It receives three parameters.
+   */
+  onClosingTab: PropTypes.func,
```

Comment:
Can we not have 1 prop instead of 2 if both are callbacks triggered on close of tab?

Reply:
The onClosingTab function determines the active selected tab in the tabs .
The activeAfterClosed state variable holds the key of the tab to become active after closure of activetab.
When in the case of Active tab is closed:The onClosingkey handle the case and updates the activeKey state according to requirement.